presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,54 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Fix a null pointer exception during the stats collection phase in query planning.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
+  when ``join_distribution_type`` was set to ``AUTOMATIC``.
+* Fix equality inference for ``VARCHAR`` columns.  This improves performance for queries that have
+  both constant predicates and join predicates on a ``VARCHAR`` column.
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns the position of the N-th instance of a substring from the back of a string.
+* Add aggregation function :func:`entropy`.
+* Add classification aggregation functions :func:`classification_miss_rate`, :func:`classification_precision`,
+  :func:`classification_recall`, :func:`classification_thresholds`.
+* Add maximum standard error to :func:`approx_set`.
+* Add ``max_tasks_per_stage`` session property and ``stage.max-tasks-per-stage`` config property to
```

Comment:
Would we expect an administrator of Presto to ever set this?

Reply:
@wenleix can you answer Tim's question?  And if the answer is yes, do we have guidance anywhere about how/why to set this configuration?

Reply:
@tdcmeehan : Yes. This can potentially help resource management (e.g. don't have to always use all the nodes for large batch queries, don't use all nodes for grouped execution, etc).

@rschlussel : We can just explain the purpose of this session property, e.g. 
```
Setting this session property allows large batch queries not always use all the nodes in the cluster for table scan, and use a fixed number of nodes to have more predictable memory usage for grouped execution, albeit the physical cluster size. 
```

Reply:
ok, i updated the release note with more info based on @wenleix's description.  @tdcmeehan @wenleix, what do you think of the new language?

Reply:
Looks good to me!